care/emr/tagging/base.py

```diff
@@ -119,12 +121,12 @@ def get_tag_config_object(self, external_id, facility=None):
 
 class PatientFacilityTagManager(SingleFacilityTagManager):
     def __init__(self, facility) -> None:
-        if isinstance(facility, str):
+        if isinstance(facility, UUID):
```

Comment:
```suggestion
        if isinstance(facility, (str, UUID)):
```

there are cases where the uuid may be passed as a string